integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -41,9 +41,10 @@ protected SparkSession getSparkSession() {
     return sparkSession;
   }
 
-  @BeforeAll
-  void startUp() {
```

Comment:
I prefer the way the base or common module defines the overall logic, the subclass could override specific methods  like `doA()` not `startup()` 
```java
void startup(){
doA()
doB()  
}
```

Reply:
SparkEnvIT could provide getProvider() and getCatalogName() method which are abstract and should be override be concrete SparkXXCatalogIT

Reply:
fixed.